livekit-ffi/src/server/room.rs

```diff
@@ -238,19 +238,21 @@ impl RoomInner {
         .to_vec();
 
         let kind = publish.kind();
-        let destination_sids = publish.destination_sids;
+        let destination_identities = publish.destination_identities;
         let async_id = server.next_id();
 
         if let Err(err) = self.data_tx.send(FfiDataPacket {
             payload: DataPacket {
                 payload: data.to_vec(), // Avoid copy?
-                kind: kind.into(),
+                reliable: match kind {
+                    proto::DataPacketKind::KindLossy => false,
```

Comment:
Let's also change the PublishData request to take a boolean as input?
Breaking the protocol is OK across FFI version. (The binaries are shipped with the SDKs, so we can break stuff safely)